From 9ec84ef1d6b4194721df70e144eadba73811d453 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Mon, 15 Jul 2024 09:44:54 +0200
Subject: [PATCH 1061/1695] dts: stm32: default enable TAMP button on
 stm32mp257f-dk board

Default enable TAMP button on stm32mp257f-dk board. The tamper is in
confirmed mode.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git ce804f3d5980e3f997236a4ad2c287f186dd434d dts: stm32: default enable TAMP button on stm32mp257f-dk board]
Change-Id: I2a894c0e1e5e3f0012f2cd688f6b93941ff77638
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/393231
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/dts/stm32mp257f-dk.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp257f-dk.dts b/core/arch/arm/dts/stm32mp257f-dk.dts
index bf3e4ebb3..2f2cf6064 100644
--- a/core/arch/arm/dts/stm32mp257f-dk.dts
+++ b/core/arch/arm/dts/stm32mp257f-dk.dts
@@ -464,7 +464,7 @@
 
 	/* Tamper button */
 	tamp-button {
-		status = "disabled";
+		status = "okay";
 		tamper-gpios = <&gpioz 2 0>;
 		st,tamp-mode = <TAMPER_CONFIRMED_MODE>;
 		st,tamp-id = <3>;
-- 
2.47.2

